setup-environment-internal

```diff
@@ -277,8 +277,11 @@ INHERIT += "buildhistory"
 BUILDHISTORY_COMMIT = "1"
 EOF
 
-# we don't need this anymore
-unset LMP_VERSION_CACHE
+# if this don't runs from CI
+if [ ! -v LMP_VERSION ]; then
```

Comment:
Does it mean that we need to set `LMP_VERSION` variable to be set so we can use sstate-cache, which version is provided in `LMP_VERSION_CACHE`?

Also, IMO, we have a confusion in naming, currently we specify version number in `LMP_VERSION_CACHE`, but `LMP_VERSION` is just boolean value, IIUC

Reply:
> Does it mean that we need to set LMP_VERSION variable to be set so we can use sstate-cache, whose version is provided in LMP_VERSION_CACHE var?

no, LMP_VERSION only exist on CI and was on of the variables that I chose to see where the execution is taking place. CI vs local

in reality they have the same value but they need to be different because the second can be increased
https://github.com/foundriesio/ci-scripts/blob/master/helpers.sh#L115-L116

Reply:
I added a more descriptive comment

Reply:
thanks!